pkg/install/install_test.go

```diff
@@ -103,23 +102,33 @@ var _ = Describe("Install", Label("install"), func() {
 		t = &transmock.Transactioner{}
 		runner = sysmock.NewRunner()
 		mounter = sysmock.NewMounter()
-		fs, cleanup, err = sysmock.TestFS(nil)
+		syscall = &sysmock.Syscall{}
+
+		fs, cleanup, err = sysmock.TestFS(map[string]any{
+			"/etc/config.sh":  []byte{},
+			"/dev/device":     []byte{},
+			"/dev/device1":    []byte{},
+			"/dev/device2":    []byte{},
+			"/dev/pts/empty":  []byte{},
+			"/proc/empty":     []byte{},
+			"/sys/empty":      []byte{},
+			"/opt/config.sh":  []byte{},
+			"/opt/tree/empty": []byte{},
+		})
 		Expect(err).ToNot(HaveOccurred())
 		s, err = sys.NewSystem(
 			sys.WithMounter(mounter), sys.WithRunner(runner),
 			sys.WithFS(fs), sys.WithLogger(log.New(log.WithDiscardAll())),
+			sys.WithSyscall(syscall),
 		)
 		Expect(err).NotTo(HaveOccurred())
-		Expect(vfs.MkdirAll(fs, "/dev", vfs.DirPerm)).To(Succeed())
-		Expect(vfs.MkdirAll(fs, "/etc", vfs.DirPerm)).To(Succeed())
-		Expect(fs.WriteFile("/dev/device", []byte{}, vfs.DirPerm)).To(Succeed())
-		Expect(fs.WriteFile("/dev/device1", []byte{}, vfs.DirPerm)).To(Succeed())
-		Expect(fs.WriteFile("/dev/device2", []byte{}, vfs.DirPerm)).To(Succeed())
 		d = deployment.DefaultDeployment()
 		d.Disks[0].Device = "/dev/device"
 		d.Disks[0].Partitions[0].UUID = "34A8-ABB8"
 		d.Disks[0].Partitions[1].UUID = "34a8abb8-ddb3-48a2-8ecc-2443e92c7510"
 		d.SourceOS = deployment.NewDirSrc("/some/dir")
+		d.CfgScript = "/opt/tree"
```

Comment:
Just for the sake of the example, the script should reside outside of the overlay tree, right?